### PR TITLE
 New add one storage cases, add one scsi disk with largest disk size

### PR DESCRIPTION
**Pull request by dulili**

### CODE REVIEW

File: testscripts/stor_largest_size.ps1 [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+#
+# Main Body
+#
+###############################################################################
+
+$retVal = $Failed
+
+# Get the VM
+$vmObj = Get-VMHost -Name $hvServer | Get-VM -Name $vmName
+#hot add two scsi disk
```
Comment (yangboroy) [inline testscripts/stor_largest_size.ps1]: second, right?

File: testscripts/stor_largest_size.ps1 [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+
+# Check the disk number of the guest.
+$diskList =  Get-HardDisk -VM $vmObj
+$diskLength = $diskList.Length
+
+if ($diskLength -eq 2)
+{
+    write-host -F Red "The disk count is $diskLength."
+    Write-Output "Add two disk successfully, The disk count is $diskLength."
```
Comment (yangboroy) [inline testscripts/stor_largest_size.ps1]: Need a check about second disk size? 
For example, add disk successfully, but size gap is large

File: testscripts/stor_largest_size.ps1 [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+# Main Body
+#
+###############################################################################
+
+$retVal = $Failed
+
+# Get the VM
+$vmObj = Get-VMHost -Name $hvServer | Get-VM -Name $vmName
+#hot add two scsi disk
+$disk = New-HardDisk -CapacityGB 63488 -VM $vmObj -StorageFormat "Thin" -ErrorAction SilentlyContinue
```
Comment (yangboroy) [inline testscripts/stor_largest_size.ps1]: I thinks need a comment about 63488 GB. For example, this is largest support, right?

Comment (dulili) [reply testscripts/stor_largest_size.ps1]: one disk, total is two, I will change it.

File: testscripts/stor_largest_size.ps1 (hunk 2 above)
Comment (dulili) [reply testscripts/stor_largest_size.ps1]: if add disk command successfully, I think the disk size should not wrong.
Also I think check the size at the end is reasonable.

**Comment by yangboroy** [comment]
Will merge them in this version. New comments will be consider in next version as current situation

**Pull request merged**